example/hybrid-mbr.nix

```diff
@@ -0,0 +1,62 @@
+{ disks ? [ "/dev/sda" ], ... }:
+{
+  disk = {
+    main = {
+      type = "disk";
+      device = builtins.elemAt disks 0;
+      content = {
+        type = "hybrid_table";
+        efiGptPartitionFirst = false;
+        hybrid_partitions = [
+          {
+            type = "hybrid_partition";
+            gptPartitionNumber = 1;
+            mbrPartitionType = "0x0c";
```

Comment:
what are other valid mbrPartitionType options?

Reply:
whatever hexcode dos table partitions support, as in https://en.wikipedia.org/wiki/Partition_type#List_of_partition_IDs.

If `null` then it guess partition type based on GPT one.

Reply:
but would it still be a hybrid mbr if the partitionType is not "0x0c"?

Reply:
Sure, set to `0x0c`have sense for RPi because RPi boot process look exactly for it, but other use cases could work with default value.